fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -405,7 +406,7 @@ void flakyMerge(boolean isGrouped,
                     assertFalse(requireFailure && i < 15, i + " merge should have failed");
                     success = true;
                 } catch (RecordCoreException e) {
-                    final LoggableTimeoutException timeoutException = findTimeoutException(e);
+                    final LoggableKeysAndValues<? extends Exception> timeoutException = findTimeoutException(e);
```

Comment:
If the new `asyncToSync` is changed to throw the same exception, this should be changed back.